app/src/main/res/values/strings.xml

```diff
@@ -320,6 +320,7 @@
     <string name="two_factors_otp_error">Two factors OTP is required</string>
     <string name="no_issue">No issue</string>
     <string name="copy">Copy</string>
+    <string name="copy_repo_url">Copy repo URL</string>
```

Comment:
yes, lets stick to copy, maybe you could change the original one to Copy URL which then will make more sense for all :)